tests/scaletest/scale_test.go

```diff
@@ -161,7 +282,30 @@ func PoolVerification(t *testing.T) bool {
 	return false
 }
 
+/* Verifies if all requires DNS A records are created in the DNS VS or not */
+func DNSARecordsVerification(t *testing.T, hostNames []string) bool {
+	t.Logf("Verifying DNS A Records...")
+	FQDNList := lib.FetchDNSARecordsFQDN(t, dnsVSUUID, AviClients[0])
+	diffString := DiffOfLists(FQDNList, hostNames)
+	if len(diffString) == initialNumOfFQDN {
+		return true
```

Comment:
Should we also check the contents of FQDNList? As in, the difference might be fine, but the contents might be different? @monotosh-avi @RiyaGaniga-avi

Reply:
@hemantavi yeah I also felt the same, but couldn't find a condition where this won't work :-)